scripts/health_metrics/README.md

```diff
@@ -35,3 +38,31 @@ pod-lib-lint-newsdk:
 Add the job name to the [`needs` of `create_report` job](https://github.com/firebase/firebase-ios-sdk/blob/64d50a7f7b3af104a88f9c9203285ae20ea309d4/.github/workflows/test_coverage.yml#L277).
 
 4. If this newsdk podspec has unit test setup, e.g. [database](https://github.com/firebase/firebase-ios-sdk/blob/64d50a7f7b3af104a88f9c9203285ae20ea309d4/FirebaseDatabase.podspec#L44-L57), with `unit_tests.scheme = { :code_coverage => true }`, the code coverage workflow should run unit tests through podspecs and utilize those coverage data and nothing is needed to update here. Otherwise, add [another way of running tests](https://github.com/firebase/firebase-ios-sdk/blob/64d50a7f7b3af104a88f9c9203285ae20ea309d4/scripts/health_metrics/pod_test_code_coverage_report.sh#L26) and generating xcresult bundles with code coverage data in pod_test_code_coverage_report.sh.
+
+## Binary Size Report Generation
+
+This tool is to generate a report for SDK binary sizes.
+[create_binary_size_report.sh](https://github.com/firebase/firebase-ios-sdk/blob/master/scripts/health_metrics/create_binary_size_report.sh)
+will trigger [BinarySizeReportGenerator](https://github.com/firebase/firebase-ios-sdk/tree/77dae88bb9784fa3ba4be21fb88972ed2c1dd64f/scripts/health_metrics/generate_code_coverage_report/Sources/BinarySizeReportGenerator)
```

Comment:
Can we refer to `master` instead of a commit in the link? So that this doc is always up to date with the source code. For example, the code you linked here still has `type` equals to `firebase-ios-sdk-testing`.